server/dive_tasks/tasks.py

```diff
@@ -450,8 +455,8 @@ def cleanup():
     folderData = gc.getFolder(folderId)
     requestedFps = fromMeta(folderData, FPSMarker)
 
-    # Extract metadata
-    file_name = os.path.join(path, os.listdir(path)[0])
+    file_name = download_source_media(gc, folderId, Path(input_directory.name))[0]
```

Comment:
Additional future thing, now that we have some concise endpoints for generating a CSV and downloading the source media, there is a recursive download items in this folder that can probably be factored out for running pipelines.  Just grabbing media and the current CSV I think is all that should be needed.